__init__.py

```diff
@@ -12,31 +12,178 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+from mycroft.skills.core import MycroftSkill
+
+######################################################################
+# Temporary, will move to mycroft.skills.core
+
+from enum import Enum
+from abc import ABC, abstractmethod
+from mycroft.skills.audioservice import AudioService
+from mycroft.messagebus.message import Message
+
+
+class CPSMatchLevel(Enum):
+        EXACT = 1
+        MULTI_KEY = 2
+        TITLE = 3
+        ARTIST = 4
+        CATEGORY = 5
+        GENERIC = 6
+
+
+class CommonPlaySkill(MycroftSkill, ABC):
+    def __init__(self, name=None, bus=None):
+        super().__init__(name, bus)
+        self.audioservice = None
+
+    def initialize(self):
```

Comment:
I changed this in the spotify skill to override bind instead so the user don't have to call super if they have an initialize method:

```python
    def bind(self, bus):
        if bus:
            super().bind(bus)
            self.audioservice = AudioService(self.bus)
            self.add_event('play:query', self.__handle_play_query)
            self.add_event('play:start', self.__handle_play_start)
```